Documentation/ceph-cluster-crd.md

```diff
@@ -985,6 +985,67 @@ If the previous section has not been completed, the Rook Operator will still ack
 
 > **WARNING**: If no cluster is managed by the current Rook Operator, you need to inject `common.yaml`, then modify `cluster-external.yaml` and specify `rook-ceph` as `namespace`.
 
+If this is successfull you will see the CepCluster status as connected.
+
+```console
+kubectl get CephCluster -n rook-ceph-external
+NAME                 DATADIRHOSTPATH   MONCOUNT   AGE    STATE       HEALTH
+rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
+```
+
+Before you create a StorageClass with this cluster you will need to create a Pool in your external Ceph Cluster.
+
+#### Example StorageClass based on external Ceph Pool
+
+In Ceph Cluster let us list the pools available
+
+```console
+rados df
```

Comment:
I'd use `ceph osd lspools`

Reply:
Yes this is also fine; What I wanted was to check if the PV created was writing to external cluster, as last time I tested it was in rook-ceph and I thought all was well

After creation you will see a bump in size, and after some data is written
```
[root@ceph-1 ~]# ssh -i alex_ee.pem ceph-2 rados df
POOL_NAME        USED OBJECTS CLONES COPIES MISSING_ON_PRIMARY UNFOUND DEGRADED RD_OPS      RD WR_OPS     WR USED COMPR UNDER COMPR 
replicated_2g 151 MiB      27      0     81                  0       0        0    137 2.2 MiB     93 50 MiB        0 B         0 B 
```

Reply:
Works for me!